mathquill4quill.css

```diff
@@ -47,7 +47,7 @@
   display: flex;
   flex-direction: column;
   align-items: center;
-  width: 300px;
+  width: 318px;
```

Comment:
Why does the width need to change? I’m coming at this from a backwards compatibility perspective and wouldn’t want to cause spurious UI changes to folks. If it’s needed for proper spacing of the delete button, consider margins on the button itself.